docs/did-button-branding/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: "doc"
+title: "Branding DID.app buttons."
```

Comment:
Maybe try: 'Sign in with DID.app' buttons for your website

Reply:
not sure. this is a part of the docs page so can be a bit more formal, also I copied that title off the sign in with google docs

Reply:
ok :-)  I think it's fine.  first time I read it was like what do you mean branding DID.app buttons.

is it Branding. DID.app buttons..

as in subjject: BRanding.

Content: DID.app buttons.

Just felt weird.  

Not overly worried.  Time is of the essence, choose one, implement it.